test/Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests/KafkaLanguageEndToEndTests.cs

```diff
@@ -16,35 +16,38 @@ public KafkaLanguageEndToEndTests(KafkaLanguageEndToEndFixture endToEndTestFixtu
             this.endToEndTestFixture = endToEndTestFixture;
         }
 
-        [Fact]
-        public async Task Java8_Smoke_Test_For_Output_And_SingleTrigger()
-        {
-            var random = new Random();
-            string inputMessage = $"hello.{random.Next(0, 9999)}";
+        //[Fact]
```

Comment:
This change shouldn't lie in the scope of this PR

Reply:
These older tests require us to create docker images with the older versions of the function apps. Building all the apps together is causing the memory on the pipeline to run out [error link](https://azfunc.visualstudio.com/Azure%20Functions/_build/results?buildId=71139&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=dee4e6ec-be1b-5671-4678-ab293b4004c5&l=42906).

Reply:
As discussed offline, we can revert the change for now